invenio_vocabularies/contrib/affiliations/mappings/os-v1/affiliations/affiliation-v2.0.0.json

```diff
@@ -0,0 +1,169 @@
+{
+  "settings": {
```

Comment:
since it is impossible to get a diff with the previous version, could you just summarise to which fields you've applied the changes?

Reply:
These are the changes for the affiliations vocabulary
```diff
@@ -6,7 +53,9 @@
  "mappings": {
    "dynamic_templates": [
      {
        "i18n_title": {
           "path_match": "title.*",
           "match_mapping_type": "string",
           "mapping": {
-            "type": "search_as_you_type"
+            "type": "search_as_you_type",
+            "analyzer": "accent_analyzer",
+            "search_analyzer": "accent_analyzer"
           }
         }
       }
@@ -40,18 +89,26 @@
       },
       "name": {
         "type": "text",
+        "analyzer": "accent_edge_analyzer",
+        "search_analyzer": "accent_edge_analyzer",
         "copy_to": "name_sort",
         "fields": {
           "suggest": {
-            "type": "search_as_you_type"
+            "type": "search_as_you_type",
+            "analyzer": "accent_analyzer",
+            "search_analyzer": "accent_analyzer"
           }
         }
       },
       "acronym": {
         "type": "text",
+        "analyzer": "accent_edge_analyzer",
+        "search_analyzer": "accent_edge_analyzer",
         "fields": {
           "suggest": {
-            "type": "search_as_you_type"
+            "type": "search_as_you_type",
+            "analyzer": "accent_analyzer",
+            "search_analyzer": "accent_analyzer"
           }
         }
       },
```